plugins/pulp_rpm/plugins/importers/yum/purge.py

```diff
@@ -263,3 +264,28 @@ def get_remote_units(file_function, tag, process_func):
     finally:
         file_handle.close()
     return remote_named_tuples
+
+
+def remove_unit_duplicate_nevra(unit_key, type_id, repo_id):
+    """
+    Removes from the repo units that have same NEVRA.
```

Comment:
I'd reword this to "Removes units from the repo that have the given unit_key, ignoring the checksum and checksum type."